.github/workflows/sycl-linux-run-tests.yml

```diff
@@ -86,6 +86,9 @@ on:
       reset_intel_gpu:
         type: string
         required: False
+      is_icx:
```

Comment:
NIT: Instead of `is_icx`, would it be better to name this variable something like `use_oneapi`?

Reply:
sure, will do

Reply:
removed this param